internal/jobworker/cgroup/cgroup_test.go

```diff
@@ -0,0 +1,233 @@
+package cgroup
+
+import (
+	"bufio"
+	"errors"
+	"io/fs"
+	"os"
+	"os/exec"
+	"path"
+	"reflect"
+	"testing"
+)
+
+func TestCleanup(t *testing.T) {
+	if !isRoot() {
+		t.Skip("must be root to run")
+	}
+
+	dir := t.TempDir()
+	service, err := NewService(WithMountPath(dir))
+	if err != nil {
+		t.Errorf("%+v", err)
+		return
```

Comment:
```suggestion
		t.Fatalf("%+v", err)
```

Reply:
Replaced all occurrences of ...
```
t.Errorf(...)
return
```
with `t.Fatalf(...)`